api/swagger/swagger.yaml

```diff
@@ -147,31 +147,50 @@ paths:
           description: Error
           schema:
             $ref: "#/definitions/ErrorResponse"
-  /event/{page},{pageSize}:
-      x-swagger-router-controller: EventController
-      get:
-        description: get a page of events
-        operationId: getEvents
-        parameters:
-          - name: page
-            type: number
-            in: path
-            required: true
-          - name: pageSize
-            type: number
-            in: path
-            required: true
-            default: 10
-        responses:
-          "200":
-            description: Success
-            schema:
-              $ref: "#/definitions/EventListResponse"
-          # responses may fall through to errors
-          default:
-            description: Error
-            schema:
-              $ref: "#/definitions/ErrorResponse"
+  /events/:
+    x-swagger-router-controller: EventController
+    get:
+      description: get pagenated events within a radius or by created time if user location is not provided
+      operationId: getEvents
+      parameters:
+        - name: sortBy
+          type: string
+          in: header
+          default: ""
+          description: order by distance by default, can also be "-createdAt" if we wanna order by created time.
+        - name: page
+          type: number
+          in: header
+          required: true
+          default: 1
+        - name: pageSize
+          type: number
+          in: header
+          required: true
+          default: 10
+        - name: lat
+          type: number
+          in: header
+          description: maybe we should default it to user current location ??
```

Comment:
we can't default it from backend, the logic is in frontend

Reply:
cool